.github/workflows/golang-ci-workflow.yaml

```diff
@@ -14,47 +14,11 @@ on:
 
 jobs:
   test:
-    name: Test with coverage
+    name: Test with coverage and send coverage report to coveralls
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: "${{ inputs.GO_VERSION }}"
-      - name: Check out code
-        uses: actions/checkout@v3
-      - name: Install dependencies
-        run: |
-          go mod download
-      - name: Lint source code
-        uses: golangci/golangci-lint-action@v3
-        with:
-          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.45
-
-          # Optional: working directory, useful for monorepos
-          # working-directory: somedir
-
-          # Optional: golangci-lint command line arguments.
-          args: --timeout=5m
-
-          # Optional: show only new issues if it's a pull request. The default value is `false`.
-          # only-new-issues: true
-
-          # Optional: if set to true then the all caching functionality will be complete disabled,
-          #           takes precedence over all other caching options.
-          skip-cache: true
-
-          # Optional: if set to true then the action don't cache or restore ~/go/pkg.
-          # skip-pkg-cache: true
-
-          # Optional: if set to true then the action don't cache or restore ~/.cache/go-build.
-          # skip-build-cache: true
-      - name: Run Unit tests
-        run: |
-          go test ${{ inputs.TESTS_LOCATION }} -v -coverpkg=${{ inputs.TESTS_LOCATION }} -race -covermode atomic -coverprofile=./profile.cov
-      - name: Install goveralls
-        run: go install github.com/mattn/goveralls@latest
+      - name: Checkout code / lint code / install dependencies for goveralls
+        uses: uc-cdis/.github/.github/actions/golang-ci@master
```

Comment:
following [this example](https://stackoverflow.com/questions/59259783/how-to-combine-two-action-yml-files-in-one-repo), perhaps we could also write this as:
```
uses: ../actions/golang-ci
```

Reply:
it would make it more readable... 🤔